README.md

```diff
@@ -19,3 +19,12 @@ MiSeq_runs_basespace.sh
 ## daily_SARS-CoV-2_metadata.sh
 
 daily_SARS-CoV-2_metadata.sh checks the date every 4 hours. If it is a new day, this script runs the Dripping Rock nextflow workflow to get the daily SARS-CoV-2 metadata file. On Sundays, it also takes the latest 20 runs and 10 random runs and creates a phylogenetic tree. This is a script that is run in a screen on the Production account.
+
+ 
+## RCovidSummary.R
+
+This script shold be run after the analytics phase and the pangolin-lineage are completed. The script summaryzes the run and generated two files: RunParameters and RCovidseq_summary, that will be stored at the runpath/Rsumcovidseq folder. 
+
+RunParameters: Is the collection of the run static parameters: Wetlab assay, instrument type, software, pangolin version, Read Type, Number of Samples, number of Lanes, Chemistry , index adapters and the date of the run.
+
+Rcovidseq_summary: Is the collecton of the dynamic variables: sample_IDs, lineage, scorpion_call, num_actg, num_n, pass/fail.
```

Comment:
I was looking at your readme, and I was wondering why it was so different than mine. Then I realized that https://github.com/UPHL-BioNGS/General_LW_scripts/pull/12/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 hasn't been merged, yet. Ooops!

I don't actually understand how or when to run this script, but here's my best guess (please change where applicable)

"""
### RCovidSummary.R

RCovidSummary.R should be run once the [dragen-covid-pipeline.sh](https://github.com/UPHL-BioNGS/dragen_scripts) has finished on a SARS-CoV-2 covidseq run (runs on the dragen server), and pangolin has been run on the resultant fasta files. The script summarizes the run and generates two files in the $runpath/Rsumcovidseq directory: RunParameters-$date.csv (values that are the same for each sample) and RCovidseq_summary-$date.csv (values that are specific to each sample)

```
Rscript RCovidSummary.R UT-A01290-220901
```

"""

Reply:
That is better explanation to know how/when the script should be run.